packages/plugins/Zoom/src/components/ZoomSlider.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div>
+    <v-slider
+      v-model="zoomLevelRange"
+      vertical
+      class="polar-zoom-slider"
+      :min="minimumZoomLevel"
+      :max="maximumZoomLevel"
+      :aria-label="$t('common:plugins.zoom.slider')"
+    ></v-slider>
+  </div>
```

Comment:
```suggestion
  <v-slider
    v-model="zoomLevelRange"
    vertical
    class="polar-zoom-slider"
    :min="minimumZoomLevel"
    :max="maximumZoomLevel"
    :aria-label="$t('common:plugins.zoom.slider')"
  ></v-slider>
```

As written in https://github.com/Dataport/polar/pull/138#discussion_r1666540302